pkg/dev/dev.go

```diff
@@ -31,12 +33,14 @@ func NewDevClient(
 	prefClient preference.Client,
 	portForwardClient portForward.Client,
 	watchClient watch.Client,
+	bindingClient binding.Client,
```

Comment:
I don't see us making use of binding client in dev pkg. Not in this commit at least.